### ISSUE TITLE
How to connect do fresh flashed device ?

### ISSUE THREAD

**Issue by 0xFACE** [aanon4/BlueBasic#19]
Hello, I successfully flashed BLE HM-10 module with BlueBasic-CC2541.hex file.
But now I don't how to connect. 
I tried to scan for the BLE devices, but HM-10 module is not advertising any more.
Is it ok ? Can I use RX/TX pins to connect console?

Unfortunately I don't have IOS/ OSX device to use the  BlueBasicConsole from your repo.

**Comment by crushedice2000** [comment]
I have the same problem with a <s>counterfeit</s> non-original HM-10.  Also there is a closed issue with the same problem: https://github.com/aanon4/BlueBasic/issues/10

**Comment by 0xFACE** [comment]
Tanks for the comment. I flashed another HM-11 module (smaller HM-10 version). 
It's advertising with "BASIC#7F". Now I can connect it with my phone and browse services.
So, it seems to be hardware issue with non-original HM-10 module (at first look one XTAL is missing comparing pictures). 

But still can't run any BASIC program.  What console to use for non IOS/ OSX OS'es ?

**Comment by crushedice2000** [comment]
@0xFACE: In theory you could use one of these apps on Android (untested because I only have a non-original one):

MSMBLE
B-BLE
BLE Scanner
Bluetooth LE Scanner

**Comment by crushedice2000** [comment]
@aanon4 (the main culprit ;-): Do you switch to the missing XTAL once the program begins? I can't figure out why the non-original HM-10 does extrange things... It starts advertising on power-up or reset and then stops/crashes... Could you test one of these?

**Comment by 0xFACE** [comment]
@crushedice2000
Yes, I can connect with these apps. But to write basic code I have to know which service(s) to use and how.

**Comment by crushedice2000** [comment]
@0xFACE: There is a list of services. You can try them. However MSMBLE highlights in green/yellow the correct service.

**Comment by 0xFACE** [comment]
I didn't find any working software for Android, so I wrote small script to upload basic file from Linux:
https://github.com/0xFACE/BlueBasic-loader

**Comment by crushedice2000** [comment]
@0xFACE: Please check my PR.
![https://assets-cdn.github.com/images/icons/emoji/unicode/1f608.png](https://assets-cdn.github.com/images/icons/emoji/unicode/1f608.png)
